flags/eigendaflags/cli.go

```diff
@@ -163,10 +181,96 @@ func CLIFlags(envPrefix, category string) []cli.Flag {
 			EnvVars:  []string{withEnvPrefix(envPrefix, "PUT_RETRIES")},
 			Category: category,
 		},
+	},
+		v2Flags(envPrefix, category)...,
+	)
+}
+
+func v2Flags(envPrefix, category string) []cli.Flag {
+	return []cli.Flag{
+		// EigenDA V2 specific flags //
+		&cli.BoolFlag{
+			Name:     V2Enabled,
+			Usage:    "Enable blob dispersal and retrieval against EigenDA v2 protocol",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "V2_ENABLED")},
+			Required: false,
+		},
+		&cli.StringFlag{
+			Name:     CertVerifierAddrName,
+			Usage:    "Address of the EigenDABlobVerifier contract. Required for performing eth_calls to verify EigenDA certificates.",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "CERT_VERIFIER_ADDR")},
+			Category: category,
+			Required: false,
+		},
+		&cli.DurationFlag{
+			Name:     RelayTimeoutName,
+			Usage:    "Timeout used when querying a relay for blob contents.",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "RELAY_TIMEOUT")},
+			Value:    10 * time.Second,
+			Required: false,
+		},
+		&cli.DurationFlag{
+			Name:     ContractCallTimeoutName,
+			Usage:    "Timeout used when performing smart contract eth_calls",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "CONTRACT_CALL_TIMEOUT")},
+			Value:    10 * time.Second,
+			Required: false,
+		},
+		&cli.UintFlag{
+			Name:     BlobVersionName,
+			Usage:    "Blob version used when dispersing. Currently only supports (0)",
```

Comment:
what is a blob version? Is this the encoding version?

Reply:
Oh the blob params in the blobHeader that shouldn't be there in the first place lol? Can you rename the flag to be more precisely named then and add more explanation as to what it is and how its used.